contracts/libs/Types.sol

```diff
@@ -70,7 +70,7 @@ library Types {
     }
 
     function committer(Batch memory batch) internal pure returns (address) {
-        return address((uint256(batch.meta) >> 80) & ADDRESS_MASK);
+        return address(uint160((uint256(batch.meta) >> 80) & ADDRESS_MASK));
```

Comment:
The `uint160` probably already includes the mask. Can't hurt to do it twice though.